src/main/java/randoop/test/RegressionCaptureGenerator.java

```diff
@@ -43,30 +46,38 @@
   private ExpectedExceptionCheckGen exceptionExpectation;
 
   /** The map from a type to the set of side-effect-free operations for the type. */
-  private MultiMap<Type, TypedOperation> sideEffectFreeMap;
+  private MultiMap<Type, TypedClassOperation> sideEffectFreeMethodsByType;
 
   /** The visibility predicate. */
   private final VisibilityPredicate isVisible;
 
+  /** The omit methods predicate to filter side effect method assertions. */
```

Comment:
Should "side effect" be "side-effect-free"?
Does "filter" mean to "remove" or to "retain"?  Avoid the ambiguous term "filter".
If this always filters side-effect-free methods, why not use a test for whether a method is side-effect-free?  (Or do you mean that this filtering is *applied* to the side-effect-free methods?  That is not clear.)